libmscore/chord.cpp

```diff
@@ -3218,7 +3218,7 @@ Element* Chord::nextElement()
                   else
                         break;
             case ElementType::ACCIDENTAL:
-                  e = e->parent();
+                  e = e->parent(); // fall through
```

Comment:
Not sure, but I beliebe that comment has to stand on ist own line.
@oberon2007?